home-data-migration/src/main/resources/db/changelog/0.0.1/Issue96.xml

```diff
@@ -0,0 +1,157 @@
+            <column name="update_date" value="2021-01-13 11:30:00"/>
+        </insert>
+    </changeSet>
+
+    <changeSet id="Issue96.003" author="Abhai2016">
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="role">
+            <column name="id" value="1"/>
+            <column name="name" value="ADMIN"/>
+        </insert>
+    </changeSet>
+
+    <changeSet id="Issue96.004" author="Abhai2016">
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="1"/>
+            <column name="name" value="GET_NEWS_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="2"/>
+            <column name="name" value="GET_USER_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="3"/>
+            <column name="name" value="CREATE_USER_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="4"/>
+            <column name="name" value="UPDATE_USER_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="5"/>
+            <column name="name" value="GET_USERS_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="6"/>
+            <column name="name" value="DELETE_USER_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="7"/>
+            <column name="name" value="UPDATE_NEWS_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="8"/>
+            <column name="name" value="CREATE_NEWS_PERMISSION"/>
+        </insert>
+        <insert dbms="postgresql"
+                schemaName="public"
+                tableName="permission">
+            <column name="id" value="9"/>
+            <column name="name" value="DELETE_NEWS_PERMISSION"/>
+        </insert>
+    </changeSet>
+
+    <changeSet id="Issue96.005" author="Abhai2016">
+        <insert  dbms="postgresql"
+                 schemaName="public"
+                 tableName="account_role">
+            <column name="role_id" value="1"/>
+            <column name="account_id" value="1"/>
+        </insert>
+    </changeSet>
+
+    <changeSet id="role_permission" author="Abhai2016">
```

Comment:
Please, use the same naming ways for all changesets ids

Reply:
Done.